src/main/java/dk/cngroup/trainings/spring/springassignment/config/WebSecurityConfig.java

```diff
@@ -28,19 +28,22 @@ protected void configure(HttpSecurity http) throws Exception {
 				.logout()
 				.logoutUrl("/web/logout")
 				.permitAll();
+
+		http.csrf().disable();
+		http.headers().frameOptions().disable();
 	}
 
 	@Override
 	public void configure(WebSecurity web) {
-		web.ignoring().antMatchers("/api/**");
+		web.ignoring().antMatchers("/api/**").and().ignoring().antMatchers("/signup");
 	}
 
 	@Bean
 	@Override
 	public UserDetailsService userDetailsService() {
```

Comment:
I think this is wrong now, you have implemented your own service, so you should not be building one here. Actually if you try to run the app and register new user, it does not work to log in via this user.

Reply:
Telling the Spring to use your service instead will probably fix that :)